CONTRIBUTORS.yaml

```diff
@@ -958,6 +958,12 @@ gatwirival:
     twitter: ValentineGatwi3
     joined: 2021-10
 
+gbiftraining:
```

Comment:
hmm, the contributors files should only be for persons. Since you already added gbif to the organisations file, would it maybe also work to add gbif under contributions->funding metadata instead of adding this as a contributor? (in any case anybody in this team who helped should be listed by name in contributors file no?)

```

Reply:
ok... I think for now I can effectively "just" remove this one and let only Sophie and Laura from GBIF ! THANK YOU

Reply:
done!